spec/sneakers/worker_handlers_spec.rb

```diff
@@ -103,6 +103,31 @@ def process(*args,&block)
 
   describe 'Maxretry' do
     let(:max_retries) { nil }
+    let(:props_with_x_death_count) {
+      {
+        :headers => {
+          "x-death" => [
+                        {
+                          "count" => 3,
+                          "reason" => "expired",
+                          "queue" => "downloads-retry",
+                          "time" => Time.now,
+                          "exchange" => "RawMail-retry",
+                          "routing-keys" => ["RawMail"]
+                        },
+                        {
+                          "count" => 3,
```

Comment:
They wouldn't have to be the same.  In practice the x-death count for both queues is always the same when it is returned to the worker queue, which is why I made them the same here.  I could change the values if you'd prefer.